### PR TITLE
Support Pypi

### PR DESCRIPTION
**Pull request by La0**
As requested in the [original PR](https://github.com/mozilla/clouseau/pull/68#issuecomment-243117988) i've ported the Pypi support code, using the new name.

You can view the package on [testpypi](https://testpypi.python.org/pypi/libmozdata)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    version = f.read().strip()
+
+with open(os.path.join(here, 'requirements.txt')) as f:
+    install_requires = f.read().strip().split('\n')
+
+
+setup(
+    name='libmozdata',
+    version=version,
+    description='Tool to find out some clues after crashes in using data from Socorro, Bugzilla and mercurial',
```
Comment (marco-c) [inline setup.py]: Can you change this to "Library to access several Mozilla data sources"?

Comment (La0) [inline setup.py]: Done!

**Comment by marco-c** [comment]
I've published it: https://pypi.python.org/pypi/libmozdata/.

**Comment by La0** [comment]
Thank you @marco-c !

**Pull request merged**